samples/openapi3/client/petstore/python-aiohttp/petstore_api/models/nullable_class.py

```diff
@@ -36,12 +36,12 @@ class NullableClass(BaseModel):
     string_prop: Optional[StrictStr] = None
     date_prop: Optional[date] = None
     datetime_prop: Optional[datetime] = None
-    array_nullable_prop: Optional[List[Union[str, Any]]] = None
-    array_and_items_nullable_prop: Optional[List[Union[str, Any]]] = None
-    array_items_nullable: Optional[List[Union[str, Any]]] = None
-    object_nullable_prop: Optional[Dict[str, Union[str, Any]]] = None
-    object_and_items_nullable_prop: Optional[Dict[str, Union[str, Any]]] = None
-    object_items_nullable: Optional[Dict[str, Union[str, Any]]] = None
+    array_nullable_prop: Optional[List[Dict[str, Any]]] = None
+    array_and_items_nullable_prop: Optional[List[Dict[str, Any]]] = None
```

Comment:
This is a bug: It must be `Optional[List[Optional[Dict[str, Any]]]]`. But I couldn't find the solution right now, maybe it can be fixed in a separate PR.

Reply:
thanks for the PR. i'll take a look over the weekend

Reply:
agreed we can address this issue later after merging this PR.

Reply:
Thanks for merging! I think the issue I see here has actually been reported already in https://github.com/OpenAPITools/openapi-generator/issues/16740.